csrankings-a.csv

```diff
@@ -1737,6 +1737,7 @@ Anne-Marie Kermarrec,EPFL,https://people.epfl.ch/anne-marie.kermarrec,aIAy-qcAAA
 Anneliese Amschler Andrews,University of Denver,http://www.cs.du.edu/~andrews,NOSCHOLARPAGE
 Anneliese Andrews,University of Denver,http://www.cs.du.edu/~andrews,NOSCHOLARPAGE
 Anneliese von Mayrhauser,University of Denver,http://www.cs.du.edu/~andrews,NOSCHOLARPAGE
+Annette Bieniusa,TU Kaiserslautern,https://softech.informatik.uni-kl.de/team/annettebieniusa,ncuwZ2cAAAAJ

```

Comment:
Need to add an entry for DBLP alias "Annette Middelkoop-Bieniusa".

Reply:
Hope I got the format right.